src/promptflow/tests/test_configs/recordings/test_flow_operations_TestFlow_test_list_flows.yaml

```diff
@@ -0,0 +1,212 @@
+        "systemData": {"createdAt": "2023-04-08T02:53:06.5886442+00:00", "createdBy":
+        "779301c0-18b2-4cdc-801b-a0a3368fee0a", "createdByType": "Application", "lastModifiedAt":
+        "2023-04-08T02:53:07.521127+00:00", "lastModifiedBy": "779301c0-18b2-4cdc-801b-a0a3368fee0a",
+        "lastModifiedByType": "Application"}}]}'
+    headers:
+      cache-control:
+      - no-cache
+      content-length:
+      - '1372'
+      content-type:
+      - application/json; charset=utf-8
+      expires:
+      - '-1'
+      pragma:
+      - no-cache
+      strict-transport-security:
+      - max-age=31536000; includeSubDomains
+      transfer-encoding:
+      - chunked
+      vary:
+      - Accept-Encoding,Accept-Encoding
+      x-content-type-options:
+      - nosniff
+      x-request-time:
+      - '0.730'
+    status:
+      code: 200
+      message: OK
+- request:
+    body: '{"filters": [{"field": "type", "operator": "eq", "values": ["flows"]},
+      {"field": "annotations/isArchived", "operator": "eq", "values": ["false"]},
+      {"field": "properties/creationContext/createdBy/userTenantId", "operator": "eq",
+      "values": ["00000000-0000-0000-0000-000000000000"]}, {"field": "properties/creationContext/createdBy/userObjectId",
+      "operator": "eq", "values": ["dccfa7b6-87c6-4f1e-af43-555f876e37a7"]}], "freeTextSearch":
+      "", "order": [{"direction": "Desc", "field": "properties/creationContext/createdTime"}],
+      "pageSize": 3, "skip": 0, "includeTotalResultCount": true, "searchBuilder":
+      "AppendPrefix"}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '614'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.31.0
+    method: POST
+    uri: https://eastus.api.azureml.ms/index/v1.0/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/00000/providers/Microsoft.MachineLearningServices/workspaces/00000/entities
+  response:
+    body:
+      string: '{"totalCount": 16, "value": [{"relevancyScore": 1.0, "entityResourceName":
+        "promptflow-eastus", "highlights": {}, "usage": {"totalCount": 0}, "schemaId":
+        "37f2e71d-b027-5d1c-a435-7d249ef7f98a", "entityId": "azureml://location/eastus/workspaceId/3e123da1-f9a5-4c91-9234-8d9ffbb39ff5/type/flows/objectId/3e123da1-f9a5-4c91-9234-8d9ffbb39ff5:7bde7cc6-fae2-4dd0-85bc-0fd6c7079a67",
+        "kind": "Unversioned", "annotations": {"archived": false, "tags": {"owner":
+        "sdk"}, "flowName": "107522a2-b3cf-4fdb-8375-a651ebe39a66", "createdDate":
+        "2023-11-15T09:07:07.5166016Z", "lastModifiedDate": "2023-11-15T09:07:07.5166017Z",
+        "owner": {"userObjectId": "dccfa7b6-87c6-4f1e-af43-555f876e37a7", "userTenantId":
+        "00000000-0000-0000-0000-000000000000", "userName": "Zhengfei Wang"}, "isArchived":
```

Comment:
Your Name

Reply:
by design for now, will sanitize name and email in the future